Upstream: yes

From eb74f7fcbf32b6ddc276401bdd6e1cd742308c89 Mon Sep 17 00:00:00 2001
From: Johannes Nixdorf <johannes@nixdorf.dev>
Date: Sat, 19 Oct 2024 13:34:54 +0200
Subject: [PATCH] Require mandir to be create before installing the man pages

Otherwise parallel installation may fail with:

 cp -f doc/kak.1 $(DESTDIR)/usr/share/man/man1
 cp: cannot create regular file '$(DESTDIR)/usr/share/man/man1': No such file or directory
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 670a48ed3610..41c0ed9e6c1d 100644
--- a/Makefile
+++ b/Makefile
@@ -201,11 +201,11 @@ install: src/kak installdirs install-debug-$(debug) install-gzip-man-$(gzip_man)
 	cp doc/pages/*.asciidoc $(sharedir)/doc
 	chmod 0644 $(sharedir)/doc/*.asciidoc
 
-install-gzip-man-yes: gzip-man-yes
+install-gzip-man-yes: gzip-man-yes installdirs
 	cp -f doc/kak.1.gz $(mandir)
 	chmod 0644 $(mandir)/kak.1.gz
 
-install-gzip-man-no: gzip-man-no
+install-gzip-man-no: gzip-man-no installdirs
 	cp -f doc/kak.1 $(mandir)
 	chmod 0644 $(mandir)/kak.1
 
-- 
2.47.0

